pkg/cmd/kafka/create/create.go

```diff
@@ -211,11 +212,11 @@ func runCreate(opts *options) error {
 		defer httpRes.Body.Close()
 	}
 
-	if apiErr := pkgKafka.GetAPIError(err); apiErr != nil {
+	if apiErr := kafkamgmt.GetAPIError(err); apiErr != nil {
 		switch apiErr.GetCode() {
-		case pkgKafka.ErrorCode24:
+		case string(kafkamgmt.ERROR_24):
 			return opts.localizer.MustLocalizeError("kafka.create.error.oneinstance")
-		case pkgKafka.ErrorCode36:
+		case string(kafkamgmt.ERROR_36):
```

Comment:
What is the namespace we can use. Is `errors` ok?

Reply:
Sounds good :+1: